tools/ratbagc.py.in

```diff
@@ -497,9 +498,38 @@ class RatbagdButton(metaclass=MetaRatbag):
 
     _PREFIX = "RATBAG_BUTTON_"
 
-    MACRO_KEY_PRESS = libratbag.RATBAG_MACRO_EVENT_KEY_PRESSED
-    MACRO_KEY_RELEASE = libratbag.RATBAG_MACRO_EVENT_KEY_RELEASED
-    MACRO_WAIT = libratbag.RATBAG_MACRO_EVENT_WAIT
+    class ActionType(IntEnum):
+        NONE = 0
```

Comment:
Shouldn't we use `libratbag.RATBAG_BUTTON_ACTION_TYPE_NONE`?

And the same for all of the following ones?

Reply:
yup